prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -252,41 +252,6 @@ presubmits: # runs on PRs
     - <<: *test_infra_ref
       base_ref: master
 
-  # TODO this is temprary job to test PR-904
-  - name: pre-master-kyma-guard-job
```

Comment:
how the guard will be executed now?

Reply:
for now, it won't be executed. If we FINALLY got approval https://github.com/kyma-project/community/pull/263 we have to execute it in every integration job.